config/live/config.yaml

```diff
@@ -9,7 +9,7 @@ deployment:
     - name: "live"
       URL: "s3://datadog-docs-live-hugo?region=us-east-1"
       exclude: "**.{go,java,sh,py,rb,json}"
-      cloudFrontDistributionID: E2B2OODXRYOXSA
+      # cloudFrontDistributionID: E2B2OODXRYOXSA
```

Comment:
I think you'll need to edit the yarn deploy live script to remove `--invalidateCDN` as well

Reply:
@zbayoff thanks, I suspected there was one more place